src/plans/stepsize.jl

```diff
@@ -872,7 +872,7 @@ mutable struct WolfePowellLinesearch{
         candidate_point::P=allocate_result(M, rand),
         candidate_tangent::T=allocate_result(M, zero_vector, candidate_point),
         candidate_direction::T=allocate_result(M, zero_vector, candidate_point),
-        max_stepsize::Real=max_stepsize(M, candidate_point),
+        max_stepsize::Real=max_stepsize(M),
```

Comment:
sure taking the global max stepsize here makes sense – however I would expect the candidate point to be a valid point so the local variant should in Principe be fine as well?

Reply:
The default value for `candidate_point` is not valid, and even if it was valid, it would make an implicit assumption that the point doesn't really matter. If it doesn't matter, why would it be passed there?

Reply:
ah because we just allocate it; I was having in mind that would be the start point or something. Then either using the start point or the current version of the global max are fine.